css/main.css

```diff
@@ -325,3 +325,86 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.mu-contact-bottom {
+  display: inline;
+  float: left;
+  width: 100%;
+}
+
+#form-messages {
+  margin-bottom: 10px;
+  background-color: #fff;
+}
+
+.mu-contact-form .form-group input {
+  background: transparent;
+  border-radius: 8px;
+  color: #333;
+  font-size: 15px;
+  border: 1px solid #888;
+  height: 45px;
+  margin-bottom: 24px;
+}
+
+.mu-contact-form .form-group textarea {
+  background: transparent;
+  border: 1px solid #888;
+  border-radius: 8px;
+  color: #333;
+  font-size: 18px;
+  padding: 10px;
+  height: 190px;
+}
```

Comment:
- [ ] *[Required]* The project specification requires you lay out the elements on the page using Flexbox but you've not made use of Flexbox at all.

Reply:
Done!